Index: src/domain/models/diagram.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># src/domain/diagram.py\r\nclass Diagram:\r\n    def __init__(self, name: str, data: dict):\r\n        self.name = name\r\n        self.data = data\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/domain/models/diagram.py b/src/domain/models/diagram.py
--- a/src/domain/models/diagram.py	(revision b8afa365e2ef4176e944270e0fa7349699453be7)
+++ b/src/domain/models/diagram.py	(date 1722155610915)
@@ -1,4 +1,5 @@
 # src/domain/diagram.py
+
 class Diagram:
     def __init__(self, name: str, data: dict):
         self.name = name
